12-transport.Rmd

```diff
@@ -261,9 +264,9 @@ od_top5 = bristol_od %>%
 od_top5 %>% 
   knitr::kable(
     caption = paste("Sample of the origin-destination data stored in the data",
-                    "frame object `bristol_od`. These represent the top 5 most", 
+                    "frame object `bristol\\_od`. These represent the top 5 most", 
```

Comment:
@Nowosad how does that look in the html?

Reply:
Good catch @Robinlovelace - it does not look perfectly... Have you got any idea how to improve it?

Reply:
Hey @Nowosad I've seen this before and there are other captions that had `_`. Would just change it to


https://github.com/Robinlovelace/geocompr/blob/b90787e016e9edda841bee159c63659c12d2d078/12-transport.Rmd#L266-L268

Reply:
Proposed fix: https://github.com/Robinlovelace/geocompr/pull/433/files